test/integration/suites/node-attestation/conf/agent/agent.conf

```diff
@@ -0,0 +1,27 @@
+agent {
```

Comment:
This config file is not needed

Reply:
will delete :)

Reply:
It seems that agent.conf is still there?